remote/src/test/scala/org/apache/pekko/remote/artery/tcp/SecureRandomFactorySpec.scala

```diff
@@ -51,7 +51,7 @@ abstract class SecureRandomFactorySpec(alg: String) extends PekkoSpec {
       "generate random" in {
         val bytes = Array.ofDim[Byte](16)
         // Reproducer of the specific issue described at
-        // https://doc.akka.io/docs/akka/current/security/2018-08-29-aes-rng.html
+        // https://pekko.apache.org/docs/pekko/current/security/2018-08-29-aes-rng.html
```

Comment:
revert this - we are not keeping the akka security docs in the pekko doc clone - so best to link this to the akka docs